source/clear-linux/tutorials/wp-install/wp-install.rst

```diff
@@ -1,215 +1,170 @@
-:ref:`installed Clear Linux on a bare metal system <bare-metal-install>`
-and that you have :ref:`set up a LAMP web server <web-server-install>`.
+Before you begin
+===========================
 
-Additionally, all the steps on this tutorial were tested using a NUC6i5SYH
-Intel® NUC. Visit the `NUC6i5SYH product page`_ for detailed information.
+This tutorial assumes you have successfully installed :ref:`Clear Linux on bare metal<bare-metal-install>` and that you have :ref:`set up a LAMP web server <web-server-install>`.
 
-Before installing any new packages, update the |CL| OS with the
-console command:
 
-.. code-block:: console
+Before you install new packages, update the |CL| OS with the following command:
+
+.. code-block:: bash
 
    sudo swupd update
 
 
-Creating a WordPress server
+Create a WordPress server
 ===========================
 
-This tutorial walks you through the process of installing the WordPress
-components on your |CL| LAMP server. At the end of the
-:ref:`web server tutorial <web-server-install>`, we created the initial
-WordPress MySQL database `WordPress` using phpMyAdmin\* and MariaDB\*.
-With the LAMP server up and running, we can add the WordPress components
-needed to host a WordPress website on your host system.
+WordPress can be installed in a variety of ways. These instructions are written for users who have followed our instructions for installing phpMyAdmin when they :ref:`set up a LAMP web server <web-server-install>`. Note that all steps in this tutorial have been tested using a NUC6i5SYH
+Intel® NUC. Visit the `NUC6i5SYH product page`_ for detailed information.
+
+Numerous online articles are available to help you name your website and acquire the necessary certificates. Those tasks are beyond the scope of this tutorial.
 
-Throughout this tutorial we will reference your website name as
-<your_website>. There are numerous articles available on-line regarding how
-to name your website and acquire the necessary certificates. Those tasks are
-beyond the scope of this tutorial.
+You can take several actions to harden your website from attacks. The security of your website and the data it contains are complex and ever-evolving tasks. Prioritize security if you plan to expose your website to the outside world. This tutorial does not address security measures that you can take to harden your site but we strongly encourage you to take action.
 
-There are also several actions you can take to harden your website from
-attacks. The security of your website and the data it contains are complex
-and ever-evolving tasks. They should be at the top of your to do list if you
-plan to expose your website to the outside world. This tutorial does not
-address security measures that you can take to harden your site but we
-strongly encourage you to look into it.
+.. note::
 
-Installing WordPress
---------------------
+   Throughout this tutorial, we reference your website name as <your_website>.
```

Comment:
Please revise the note as follows: 

.. note::
  [empty line] 
   Throughout this tutorial, ...

Note: Enter a hard return after `.. note:: `. Enter an empty line. Then press `Spacebar` to align the 'T' in Throughout with the 'n' in `note`.

Reply:
In this GitHub GUI, the formatting will not appear correctly. For another example, see the first note in `bare-metal-install.rst`

Reply:
Hi Michael. Thanks for the tip. I entered this in the file (empty line, spacebar, align) although it does not show up in the diff. Just FYI.

Reply:
The extra line and indentation for the body of the note look correct.  Nice work.